test/TestServerProjects/body-complex/Generated/Models/ArrayWrapper.Serialization.cs

```diff
@@ -5,15 +5,15 @@
 
 namespace body_complex.Models.V20160229
 {
-    public partial class ArrayWrapper
+    public partial class ArrayWrapperSerializer
```

Comment:
I think this is fine for now to have serialization be separated from the type itself (because of inheritance). However, we should figure out later on what design makes sense for serialization when dealing with derived types.

*Future thought*: I'm still thinking we have the types implement an interface for serialization. That way, we can have others create C# custom models, and have them work in our generated system, as long as those models implement that interface. It seems the most natural.